static/js/topic_page/map_tile.tsx

```diff
@@ -49,6 +49,7 @@ import { getDateRange } from "../utils/string_utils";
 import { ChartTileContainer } from "./chart_tile";
 import { CHART_HEIGHT } from "./constants";
 import { ReplacementStrings } from "./string_utils";
+export const MAP_ID = "map";
```

Comment:
The topic page is designed for multiple map tiles to load, which is why the element ID is passed in as a prop, and not hardcoded to a single ID (HTML element IDs must be unique within a page).

Please pass this value in from the disease page component.

Reply:
Done!